### PR TITLE
Add attributes check to ScopeIndentSniff

### PR DESCRIPTION
**Pull request by nlighteneddesign**

### CODE REVIEW

File: tests/Drupal/bad/bad.php [hunk 1]
```diff
@@ -838,6 +838,26 @@ function test28() {
 enum aBad_NameForAnEnum {
 }
 
+/**
+ * Test PHP attributes.
+ */
+class TestPhpAttributes {
```
Comment (klausi) [inline tests/Drupal/bad/bad.php]: please move this test to ScopeIndentUnitTest.inc and adapt the fixed file as well as the list of errors in ScopeIndentUnitTest.php.

**Review by klausi — CHANGES_REQUESTED** [review]
Thanks, please move the test.

File: tests/Drupal/bad/bad.php (hunk 1 above)
Comment (nlighteneddesign) [reply tests/Drupal/bad/bad.php]: I think I addressed both of these!

**Comment by klausi** [comment]
Thanks, tests are failing because the error lines are not updated. But I can do that in a quick follow-up.

**Pull request merged**